react-debugging.html

```diff
@@ -89,3 +93,5 @@ <h1>Draw a Card</h1>
     </script>
   </body>
 </html>
+
+<!-- Note to Instructors: Lines 66-74 are a bit problematic for me, after doing some research thru the console log and web I know I need to close it correctly but I've exhausted all my options. Not understanding the syntax formatting too well, even with cheat sheets. -->
```

Comment:
Thank you for describing the challenge you were having.  Remember that an onClick must always reference a method.  Also, will need to use `this` since it's referencing a class method.